components/TextFields/tests/unit/TextFieldTests.swift

```diff
@@ -100,10 +129,29 @@ class TextFieldTests: XCTestCase {
 
     XCTAssertTrue(textField.subviews.contains(leftView))
     XCTAssertTrue(textField.subviews.contains(rightView))
+
+    XCTAssertEqual(textField.leadingView, leftView)
```

Comment:
Should we check the orientation first? This test will fail if the Simulator is in an RTL locale, no?